content/css/concepts/position/terms/top/top.md

```diff
@@ -56,3 +60,7 @@ Set the position of `.box` element `100px` from the elements top edge.
   top: 100px;
 }
 ```
+
+![CSS Top Attribute Relative](images/rel_pos_100px.png)
```

Comment:
In a similar fashion mentioned in the suggestion above, update the `alt-text` and the image link.

Reply:
Done.